MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -763,4 +775,18 @@ - (void)hideCustomeWebViewIfNeed
     }
 }
 
+- (void) receivedGlobalSignoutDarwinNotification:(NSNotification *)notification
+{
+    self.resultTextView.text = @"Darwin notification received from the broker SDK indicating a global signout has occurred.";
+}
+
+void globalSignoutCallback(__unused CFNotificationCenterRef center,
+                           __unused void * observer,
+                           __unused CFStringRef name,
+                           __unused void const * object,
+                           __unused CFDictionaryRef userInfo)
+{
+    [[NSNotificationCenter defaultCenter] postNotificationName:kDarwinNotificationReceivedKey object:nil];
```

Comment:
Any reason why a separate NSNotificationCenter callback is required?

Reply:
The globalSignoutCallback is a C function that has a different set of rules for how it recognizes the other functions within the class. I took inspiration from one of the ways Teams uses the Darwin callback in their code for a quick way to get it working.

Reply:
can we prefix this funciton the same way as we do with other constants/global functions?